templates/repo/issue/milestones.tmpl

```diff
@@ -88,7 +88,7 @@
 							{{svg "octicon-check" 16 "gt-mr-3"}}
 							{{LocaleNumber .NumClosedIssues}}&nbsp;{{$.locale.Tr "repo.issues.closed_title"}}
 							{{if .TotalTrackedTime}}{{svg "octicon-clock"}} {{.TotalTrackedTime|Sec2Time}}{{end}}
-							{{if .UpdatedUnix}}{{svg "octicon-clock"}} {{$.locale.Tr "repo.milestones.updated"}} {{TimeSinceUnix .UpdatedUnix $.locale}}{{end}}
+							{{if .UpdatedUnix}}{{svg "octicon-clock"}} {{$.locale.Tr "repo.milestones.updated" (TimeSinceUnix .UpdatedUnix $.locale)}}{{end}}
```

Comment:
I do not think it works .... do you have a screenshot?

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/24070/commits/729f1742fc480462c4d523a130eb4535c9eaca62